packages/manager/src/features/Linodes/LinodeCreate/Networking/InterfaceType.tsx

```diff
@@ -21,14 +27,20 @@ const interfaceTypes = [
   {
     label: 'Public Internet',
     purpose: 'public',
+    description:
+      'Connects your Linode to the internet enabling traffic over the public IP address.',
```

Comment:
```suggestion
      'Connects your Linode to the internet, enabling traffic over the public IP address.',
```
I think this does need a comma.

Reply:
Yes, thank you. Updated